source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,187 @@
+---
+id: add-new-attribute
+title: Add a new attribute
+---
+
+Let's say we want to add a new element to display on a product's page (for instance, a product's rate). You will have to:
+
+1. Add the attribute's actual value to your database.
+2. Create a new module.
+3. Define the schema, implement the resolver and display it on screen.
+
+## Add a value to your database
+
+The process to do so may vary with the backend software you are using. For Magento 2 (used in this example), you can refer to [this](https://www.fastcomet.com/tutorials/magento2/product-attributes).
+
+## Create a new GraphQL module
+
+We want to display the `rate` of a product on the product page. To keep it simple, it will only consist in displaying a number, the value can be manually edited by the merchant. This feature will be stored in a new module called `mymodule` ; you could directly edit the files at `node_modules/front-commerce/path/to/something`, but this is not recommended since running `npm install` might erase any modification.
```

Comment:
```suggestion
We want to display the `rate` of a product on the product page. To keep it simple, it will only consist in displaying a number, which can be manually edited by the merchant. This feature will be stored in a new module called `mymodule`.

<blockquote class="info">Instead of creating a module, you could directly edit the files at `node_modules/front-commerce/path/to/something`. However it is highly discouraged since running `npm install` might erase any modification.</blockquote>
```

I've moved the note about the node_modules in a separate paragraph because it is a best practice in the `node` ecosystem, so it's not mandatory to mention it. Thus I make this a tiny bit more explicit by changing the information hierarchy.